.github/workflows/main.yml

```diff
@@ -60,22 +48,36 @@ jobs:
         run: |
           python kapew.py build
 
-      - name: Add package data files to dist
-        run: |
-          mv src/kolibri dist/win/Kolibri
-          mv dist/win/Kolibri kolibri-electron/src/
+      - name: Install electron-forge
+        working-directory: kolibri-electron
+        run: yarn global add @electron-forge/cli
 
       - name: Install electron dependencies
         working-directory: kolibri-electron
-        run: |
-          yarn install
-          yarn make --arch=ia32
+        run: yarn install
+
+      - name: Build electron app
+        working-directory: kolibri-electron
+        run: yarn make --arch=ia32
 
       - name: Create kolibri-windows
         run: |
           mv kolibri-electron/out/kolibri-electron-win32-ia32 kolibri-windows
 
+          mv src/kolibri dist/win/Kolibri
+          mv dist/win/Kolibri kolibri-windows/resources/app/src/
+
+      - name: Compress
```

Comment:
Yep! I agree it's a good thing to add compression. Thanks for explaining the changes in the commit message.